src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -23,6 +23,13 @@ trait DataAccess
   
   import driver.api._
 
+  lazy val createTempTables = Seq(
+    """create temporary table EXPREVAL_TEMP (id bigint(20) unsigned NOT NULL, name VARCHAR(254) NOT NULL, transaction_id CHAR(36) NOT NULL)""",
+    """create temporary table ENTITY_ATTRIBUTE_TEMP (id bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key, name text NOT NULL, value_string text, value_number double DEFAULT NULL, value_boolean bit(1) DEFAULT NULL, value_entity_ref bigint(20) unsigned DEFAULT NULL, list_index int(11) DEFAULT NULL, list_length int(11) DEFAULT NULL, owner_id bigint(20) unsigned NOT NULL, transaction_id CHAR(36) NOT NULL)""",
+    """create temporary table SUBMISSION_ATTRIBUTE_TEMP (id bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key, name text NOT NULL, value_string text, value_number double DEFAULT NULL, value_boolean bit(1) DEFAULT NULL, value_entity_ref bigint(20) unsigned DEFAULT NULL, list_index int(11) DEFAULT NULL, list_length int(11) DEFAULT NULL, owner_id bigint(20) unsigned NOT NULL, transaction_id CHAR(36) NOT NULL)""",
+    """create temporary table WORKSPACE_ATTRIBUTE_TEMP (id bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key, name text NOT NULL, value_string text, value_number double DEFAULT NULL, value_boolean bit(1) DEFAULT NULL, value_entity_ref bigint(20) unsigned DEFAULT NULL, list_index int(11) DEFAULT NULL, list_length int(11) DEFAULT NULL, owner_id binary(16) NOT NULL, transaction_id CHAR(36) NOT NULL)"""
+  ).mkString("; ")
+
```

Comment:
I'd love to see the SQL for the `*_ATTRIBUTE_TEMP` tables be centralized, not copy-pasted - but you'd have to deal with the bigint vs. binary difference for owner_id.